source/_dashboards/thermostat.markdown

```diff
@@ -38,6 +38,11 @@ theme:
   required: false
   description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
   type: string
+show_current_as_primary:
+  required: false
+  description: Show current temperature as primary information instead of target temperature. In that case, target temperature will be displayed as secondary information.
```

Comment:
```suggestion
  description: Show current temperature as primary information instead of target temperature. The target temperature will be displayed as secondary information.
```